prisma/schema.prisma

```diff
@@ -61,12 +61,23 @@ model MenuItem{
   nameOfDish String
   category String
   description String
-  image String
   price Decimal @db.Decimal(12,2)
-  restaurantId Int 
+  restaurantId String 
+  imageId Int @unique
   restaurant Restaurant @relation(fields: [restaurantId], references: [id])
   modificationGroups ModificationGroup[]   
   orderLines OrderLine[]
+  image MenuItemImage @relation(fields: [imageId], references: [id])
+}
+
+model MenuItemImage{
```

Comment:
Let's rename to Image table. We are going to reuse it for most a lot for other modules as well